### DISCUSSION TITLE
Video tutorial

### DISCUSSION THREAD

**Discussion by dustyghost66** [UltimaHoarder/UltimaScraper#1779] [Q&A]
Hi, im completely a noob, i have tried the instructions but can't seem to get it working. Could i ask if someone could make a video tutorial on how to create, install and use this? It would be much appriciated.

**Comment by fightcancer2** [comment]
Where you getting stuck?

**Reply by BEMC2001** [reply]
I get this error, I had 7.1, and everything was fine, there I read that Onlyfans became more strict and that is why the script must be updated, download the most recent and also the most recent version of python, but I get that error, I don't know if the command is wrong when I put it in the CMD, but I just copied and pasted it and it didn't work, please help me with a video tutorial
![Screenshot_20211220-183520_Telegram](https://user-images.githubusercontent.com/96454289/146856011-cd80bc3f-deac-421b-ba12-f39561c6902c.jpg)

**Comment by jozipaulo** [comment]
This is an old version of the scraper program. The latest is 7.6.1 at this time (26 Dec 2021)
You can locate the latest version here - https://github.com/DIGITALCRIMINALS/OnlyFans
Looks like this DIGITALCRIMINAL not DIGITALCRIMINALS repository is no longer active.
